src/main/scala/is/hail/sparkextras/OrderedRDD.scala

```diff
@@ -338,9 +338,35 @@ class OrderedRDD[PK, K, V] private(rdd: RDD[(K, V)], val orderedPartitioner: Ord
 
   override def getPreferredLocations(split: Partition): Seq[String] = rdd.preferredLocations(split)
 
-  def orderedLeftJoinDistinct[V2](other: OrderedRDD[PK, K, V2]): RDD[(K, (V, Option[V2]))] =
+  def groupByKey(): OrderedRDD[PK, K, Array[V]] = {
+    new OrderedRDD[PK, K, Array[V]](rdd.mapPartitions { it =>
+
+      val bit = it.buffered
+      val buffer = mutable.ArrayBuffer.empty[V]
```

Comment:
groupByKey isn't actually used anywhere in the code at the moment. I wrote this and then deleted the use in favor of something else. I think I'll just remove it